dist/api_data.py

```diff
@@ -413,9 +413,10 @@ def __cmp__(self, other):
         blocking. This option is intended for use with internal stress
         testing of WiredTiger. Only available if WiredTiger is configured
         with --enable-diagnostic. Options are given as a list, such as
-        <code>"diagnostic_timing_stress=[checkpoint_slow]"</code>''',
+        <code>"diagnostic_timing_stress=[checkpoint_slow,
```

Comment:
I'd like you to take out the dependency on diagnostic mode and update the option name and general description. When we are chasing race conditions, they sometimes only occur in non-diagnostic builds, so it's not an ideal gating flag.